packages/framer-motion/src/render/html/utils/build-styles.ts

```diff
@@ -69,12 +69,12 @@ export function buildHTMLStyles(
      * Build a transformOrigin style. Uses the same defaults as the browser for
      * undefined origins.
      */
-    if (hasTransformOrigin) {
+    if (hasTransformOrigin || style.transform) {
```

Comment:
If you move this logic to the `svg/utils/build-styles` file this will remove the style from HTML elements